crates/fj-host/src/host_handle.rs

```diff
@@ -0,0 +1,82 @@
+use std::thread::JoinHandle;
+
+use crossbeam_channel::Sender;
+
+use crate::{EventLoopClosed, Model};
+
+/// A handle for sending commands to a spawned host
+pub struct HostHandle {
+    command_tx: Sender<HostCommand>,
+    host_thread: Option<JoinHandle<Result<(), EventLoopClosed>>>,
+    model_loaded: bool,
+}
+
+impl HostHandle {
+    /// Create a `HostHandle` with a send channel and the host thread handle.
+    pub fn new(
```

Comment:
Is this part of the public API? Maybe `pub(crate)` is more appropriate?

Reply:
Thanks for pointing that - I haven't done much with API encapsulation, and although I've seen `pub(crate)' in other code, I haven't thought much about what it's for.  Now I know.

I made some bigger changes to the host API from the other conversation, and I think I got the API boundaries correct now, but not completely sure.

Reply:
To be clear, this isn't very important, in my opinion, just a small improvement. My main concern with a public constructor like this is, that a user might see that in documentation, think they're supposed to use it, then get confused about how (when actually, they should create a `Host` and get a `HostHandle` through that).

In any case, I'll take a look at your new changes later, and this conversation can be resolved.